zerver/models.py

```diff
@@ -877,7 +877,7 @@ def get_human_billing_admin_and_realm_owner_users(self) -> QuerySet["UserProfile
         )
 
     def get_active_users(self) -> QuerySet["UserProfile"]:
-        return UserProfile.objects.filter(realm=self, is_active=True).select_related()
+        return UserProfile.objects.filter(realm=self, is_active=True)
```

Comment:
I note the main other place we use this is `can_access_delivery_email(user_profile, ...)`, which does not appear to currently inspect any realm-level properties, so this should be good. But the downside of this type of change is that if in the future, if `can_access_delivery_email` were to need to access the realm, that'd start doing database queries in a loop.

One idea would be for calls like this to use https://github.com/charettes/django-seal; I'm not sure it helps but will post about it in #backend.